app/sign.js

```diff
@@ -408,7 +413,7 @@ const handleSign = function(args) {
   console.log(`Writing signed transaction to file: ${filename}`);
 
   let finalRecovery;
-  
```

Comment:
xpub6D6t5NaEsb3cMsFJN8arUShYejqVXHxQcZU73FvENS1zvXJYE7bvNXHBjDL1Dz5KgT97jxkmSGq5nRkzR8T1NKayU19x4VgsP8YLfjKCvrm

Reply:
what is this xpub for?

Reply:
oh crap, we accidentally recovered 1,043 BTC to that xpub. whoops

Reply:
can you send it back please?